stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListItemStyle.kt

```diff
@@ -92,6 +93,7 @@ public data class MessageListItemStyle(
     @Px public val messageStrokeWidthTheirs: Float,
     public val textStyleSystemMessage: TextStyle,
     public val textStyleErrorMessage: TextStyle,
+    public val messageItemTransformer: (List<MessageListItem.MessageItem>) -> Unit,
```

Comment:
Why do we need to add it to our `Style`??
It is not part of the style of this view, we should have it into the `MessageListView` itself, the same way we have `MessageListView::messageListItemPredicate`

Reply:
Moved